safe_transaction_service/history/management/commands/fix_ethereum_txs.py

```diff
@@ -1,35 +1,56 @@
+from typing import Iterator
+
 from django.core.management.base import BaseCommand
-from django.db.models import Q
+
+from gnosis.eth import EthereumClientProvider
 
 from ...models import EthereumTx
-from ...services import IndexServiceProvider
 
 
 class Command(BaseCommand):
-    help = "Check all stored ethereum_txs have a valid receipt and block. Fixes them if a problem is found"
+    help = "Fix EIP1559 transactions"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.ethereum_client = EthereumClientProvider()
+
+    def fix_ethereum_txs(self, ethereum_txs: Iterator[EthereumTx]):
+        if ethereum_txs:
+            txs = self.ethereum_client.get_transactions(
+                [ethereum_tx.tx_hash for ethereum_tx in ethereum_txs]
+            )
+            for tx, ethereum_tx in zip(txs, ethereum_txs):
+                if tx and "maxFeePerGas" in tx:
+                    self.stdout.write(
+                        self.style.SUCCESS(
+                            f"Fixing tx with tx-hash={ethereum_tx.tx_hash}"
+                        )
```

Comment:
Should we use the logger instead here?

Reply:
No, for command line scripts you should do it this way: https://docs.djangoproject.com/en/4.0/howto/custom-management-commands/